README.md

```diff
@@ -83,12 +83,49 @@ within `kafka/ConsumerPositiveComprehensiveIT.java` to cover as many of
 the [transformation rules](https://github.com/companieshouse/filing-history-delta-consumer/blob/df6ee016f916e303474034ace5c3cc346b50d441/src/main/resources/transform_rules.yml)
 as possible.
 
+### Running manual bulk integration tests
+
+The bulk integration tests uses a set of test deltas, derived from the Live CHIPS database, to check
+that they are transformed into PUT requests compatible with the documents generated by the legacy
+Perl implementation. These test cases are run manually as they depend on a local Tilt development
+environment.
+
+Tilt must have these following services enabled:
+
+```
+Manually activated modules:
+ - streaming
+ - backend
+
+Manually activated services:
+ - ch-gov-uk
```

Comment:
I think the only required manually activated service is ch-gov-uk, the rest are either not required or automatically enabled by one of the modules.